doc/source/index.rst.template

```diff
@@ -87,6 +87,7 @@ programming language.
                     <p class="card-text">Saw a typo in the documentation? Want to improve
                     existing functionalities? The contributing guidelines will guide
                     you through the process of improving pandas.</p>
+                </div>
```

Comment:
not sure why this `div` and `img` not originally indented as children of the outer `div`

Reply:
:man_shrugging:

Reply:
yeh definitely a shrug!